docs/content/doc/installation/with-docker.en-us.md

```diff
@@ -301,88 +302,66 @@ docker-compose up -d
-UID/GID as the container values `USER_UID`/`USER_GID`. You should also create the directory
-`/var/lib/gitea` on the host, owned by the `git` user and mounted in the container, e.g.
+Since SSH is running inside the container, SSH needs to be passed through from the host to the container if SSH support is desired. One option would be to run the container SSH on a non-standard port (or moving the host port to a non-standard port). Another option which might be more straightforward is to forward SSH connections from the host to the container. This setup is explained in the following.
 
-```
-version: "3"
+This guide assumes that you have created a user on the host called `git` which shares the same `UID`/ `GID` as the container values `USER_UID`/ `USER_GID`. These values can be set as environment variables in the `docker-compose.yml`:
 
-  services:
-    server:
-      image: gitea/gitea:{{< version >}}
-      container_name: gitea
-      environment:
-        - USER_UID=1000
-        - USER_GID=1000
-      restart: always
-      networks:
-        - gitea
-      volumes:
-        - /var/lib/gitea:/data
-        - /etc/timezone:/etc/timezone:ro
-        - /etc/localtime:/etc/localtime:ro
-      ports:
-        - "3000:3000"
-        - "127.0.0.1:2222:22"
+```bash
+environment:
+  - USER_UID=1000
+  - USER_GID=1000
 ```
 
-You can see that we're also exposing the container SSH port to port 2222 on the host, and binding this
-to 127.0.0.1 to prevent it being accessible external to the host machine itself.
+Next mount `/home/git/.ssh` of the host into the container. Otherwise the SSH authentication cannot work inside the container.
 
-On the **host**, you should create the file `/app/gitea/gitea` with the following contents and
-make it executable (`chmod +x /app/gitea/gitea`):
-
-```
-#!/bin/sh
-ssh -p 2222 -o StrictHostKeyChecking=no git@127.0.0.1 "SSH_ORIGINAL_COMMAND=\"$SSH_ORIGINAL_COMMAND\" $0 $@"
+```bash
+volumes:
+  - /home/git/.ssh/:/data/git/.ssh
 ```
 
-Your `git` user needs to have an SSH key generated:
+Now a SSH key pair needs to be created on the host. This key pair will be used to authenticate the `git` user on the host to the container.
 
-```
+```bash
 sudo -u git ssh-keygen -t rsa -b 4096 -C "Gitea Host Key"
 ```
 
-Now, proceed with one of the points given below:
-
-- symlink the container `.ssh/authorized_keys` file to your git user `.ssh/authorized_keys`.
-This can be done on the host as the `/var/lib/gitea` directory is mounted inside the container under `/data`:
+In the next step a file named `/app/gitea/gitea` (with executable permissions) needs to be created on the host. This file will issue the SSH forwarding from the host to the container. Add the following contents to `/app/gitea/gitea`:
 
-```
-ln -s /var/lib/gitea/git/.ssh/authorized_keys /home/git/.ssh/authorized_keys
+```bash
+ssh -p 2222 -o StrictHostKeyChecking=no [git@127.0.0.1](mailto:git@127.0.0.1) "SSH_ORIGINAL_COMMAND=\"$SSH_ORIGINAL_COMMAND\" $0 $@"
```

Comment:
I think this mailto may have snuck in unintentionally.

Reply:
Feel free to remove it - I just used what Gitea already uses in this place.

Reply:
Removed it.